packages/datatrak-web/src/views/OfflinePage.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button as UIButton } from '@tupaia/ui-components';
+import { Link as RouterLink } from 'react-router-dom';
+import { Typography } from '@material-ui/core';
+import { HEADER_HEIGHT } from '../constants';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+`;
+
+const HeaderContainer = styled.div`
+  background: ${({ theme }) => theme.palette.background.paper};
+`;
+
+const HeaderInner = styled.div`
+  display: flex;
+  align-items: center;
+  block-size: ${HEADER_HEIGHT};
+  padding: 0 1.5rem;
+  margin: 0 auto;
+  max-inline-size: 75rem;
+`;
+
+const Logo = styled(UIButton)`
+  padding: 0;
+  .MuiButton-label {
+    block-size: 2.1rem;
+
+    ${({ theme }) => theme.breakpoints.up('md')} {
+      block-size: 2.6rem;
+    }
+  }
+`;
+
+const Header = () => {
+  return (
+    <HeaderContainer>
+      <HeaderInner>
+        <Logo component={RouterLink} to="/" title="Home" variant="text">
+          <img
+            src="/datatrak-logo-black.svg"
+            alt="Tupaia Datatrak logo"
+            width="100%"
+            height="100%"
```

Comment:
Correct me if React has some special sauce, but if this is a plain ol’ vanilla `<img>`, then this must be a unitless integer. If my understanding is correct, these values as-is will be ignored

These are the intrinsic dimensions from the file:

```suggestion
            width={84}
            height={42}
```